dataprep_ml/cleaners.py

```diff
@@ -313,11 +336,13 @@ def _remove_columns(data: pd.DataFrame, identifiers: Dict[str, object], target:
     :param identifiers: A dict containing all identifier typed columns
     :param target: The target columns
     :param mode: Can be "predict" or "train"
-    :param timeseries_settings: Timeseries related settings, only relevant for timeseries predictors, otherwise can be the default object
+    :param timeseries_settings: Timeseries related settings, only relevant for timeseries predictors, otherwise can be 
+    the default object
```

Comment:
nitpick: extra newline? Should be ignored by the linter anyway, unless this somehow improves the docs formatting (in which case, it's a welcome change!)